### DISCUSSION TITLE
Stop / ramp down plotman

### DISCUSSION THREAD

**Discussion by vvavepacket** [ericaltendorf/plotman#497] [Q&A]
Say i have 7 parallel plots, how can I tell plotman to stop creating new jobs.. is there i need to tell config yaml? Or shall I set everything / global max to 0?

What happens to currently running plots if i change the config.yaml?

**Comment by cyperbg** [comment]
Global max should hopefully work.
Current jobs should finish.

**Comment by JoshuaEstes** [comment]
plotman is just a wrapper around the command "chia plots create" once a plot is kicked off, it will continue to keep going. Running "plotman interactive" will kick off new jobs if the criteria is met. You can exit out of plotman interactive or set the global max to 0. I've done both.

Also because it is just a wrapper around chia, you can add/remove tmp/dst drives and restart plotman and it will start using those. Another option I've done is that once the drive get close to full I remove it from the yaml file so no more plots are placed there. If need be, I top it off with plots from other drives.

**Reply by dragon0087** [reply]
Hi, may I ask how do you quit when running with CLI not the interactive mode? Should I enter interactive first then press q? Will that quit process I run with "plotman plot" ?

**Reply by cyperbg** [reply]
Quit what?
If you want to completely stop plotting then:
plotman kill all
otherwise: Ctrl+C or q

**Comment by syadnom** [comment]
just reinforcing what cyperbg and JoshuaEstes said.  set global max to 0 and reload interactive so you can still watch progress but no new jobs will fire.
